setup.py

```diff
@@ -132,7 +133,7 @@ def run(self):
                     shutil.rmtree(os.path.join(dirpath, dirname))
 
 
-cmdclass = {'clean': CleanCommand}
+cmdclass = {'clean': CleanCommand, 'sdist': sdist}
```

Comment:
distutils is reaching end of life and I don't think we should rely on it. 

Using
```
setuptools.command.sdist
```
here doesn't work?

Reply:
By default it's setuptools' sdist command. If I don't do that I get this warning:
```
Partial import of sklearn during the build process.
running sdist
running egg_info
/home/jeremie/miniconda/envs/dev2/lib/python3.8/distutils/dist.py:985: UserWarning: 
`build_src` is being run, this may lead to missing
files in your sdist!  You want to use distutils.sdist
instead of the setuptools version:

    from distutils.command.sdist import sdist
    cmdclass={'sdist': sdist}"

See numpy's setup.py or gh-7131 for details.
```

and then an error. See [https://github.com/numpy/numpy/pull/7131](https://github.com/numpy/numpy/pull/7131)